src/ToDoList/ToDoList.tsx

```diff
@@ -1,29 +1,46 @@
-import React from 'react';
-import {useCallback} from "react";
-import {useState} from "react";
+import React, {useCallback, useState} from 'react';
```

Comment:
please configure 'Code style' section in WebStorm settings to have next style (add spaces inside curly braces):
```suggestion
import React, { useCallback, useState } from 'react';
```

Reply:
Which must be checked?
![Снимок экрана (16)](https://github.com/AleksejC/ToDoList/assets/118013562/a85a75e0-cb1e-46a7-96ef-bb8cb1f6b15c)

Reply:
<img width="986" alt="Screenshot 2023-10-19 at 23 20 01" src="https://github.com/AleksejC/ToDoList/assets/50061172/65ff0723-716e-41d2-bcbb-90bf2415a0cf">